
#include <jni.h>
#include <android/log.h>
#define LOGI(...) __android_log_print(ANDROID_LOG_INFO, "REALITYSCAN", __VA_ARGS__)
// هنا تدمج OpenCV + COLMAP + Open3D لاحقًا
extern "C" JNIEXPORT jboolean JNICALL
Java_com_realityscan3d_app_scan_PhotogrammetryEngine_nativeBuildMesh(JNIEnv* env, jobject, jobject dense) {
    LOGI("Meshing started - Poisson Reconstruction");
    return true;
}
